com.github.bitseather.weather.json

```diff
@@ -0,0 +1,168 @@
+{
+    "app-id": "com.github.bitseater.weather",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.bitseater.weather",
+    "build-options": {
+        "cflags": "-O2  -Wno-deprecated-declarations",
+        "cxxflags": "-O2"
+    },
+    "appstream-compose": false,
```

Comment:
Correct me if I'm wrong, but doesn't that mean that no appdata will be exported correctly?

Reply:
Yeah, what I wanted is until a new release is released with the corrected appdata file, to verify if the build is successful on other architectures.